AsyncDisplayKit/Layout/ASLayoutSpec.h

```diff
@@ -25,8 +23,6 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, assign) BOOL isMutable;
 
-@property (nonatomic, strong, nullable) ASTraitCollection *traitCollection;
```

Comment:
The method `asyncTraitCollection` does exist via the `ASEnvironment` protocol. I guess it is kind of hidden and I could have kept the property as readonly.